ci/run_tests.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
 set -evx
 
-chef-server-ctl test -J $WORKSPACE/pedant.xml --all
+sudo chef-server-ctl test -J $WORKSPACE/pedant.xml --all
+
+if [ $OMNIBUS_FIPS_MODE == "true" ]
+then
+  echo "fips true" >> /etc/opscode/chef-server.rb
+  sudo chef-server-ctl reconfigure
+  echo ""
+  echo "Sleeping 120 seconds to allow the Chef Server to reconfigure in FIPS mode"
+  echo ""
+  sleep 120
+  sudo chef-server-ctl test -J $WORKSPACE/pedant.xml
```

Comment:
I'm +1 on a different file, but do we need to update the pipeline to look for both?